xenstoreclient/xenstore.go

```diff
@@ -529,7 +536,23 @@ func (xs *CachedXenStore) StopWatch() error {
 }
 
 func (xs *CachedXenStore) Clear() {
-	xs.writeCache = make(map[string]string, 0)
+	xs.writeCache = make(map[string]ValueUpdate, 0)
+}
+
+func (xs *CachedXenStore) UpdateCache() error {
```

Comment:
I think the function is not UpdateCache,
By reading the code, I think it may call: InvalidCacheFlush